plugins/inputs/opcua/README.md

```diff
@@ -0,0 +1,76 @@
+# Telegraf Input Plugin: opcua_client
+
+The opcua_client plugin retrieves data from OPCUA slave devices
+
+### Configuration:
+
+```toml
+
+# ## Connection Configuration
+#  ##
+#  ## The plugin supports connections to PLCs via OPCUA
+#  ##
+#  ## Device name
+name = "opcua_rocks"
+#
+#  # OPC UA Endpoint URL
+endpoint = "opc.tcp://opcua.rocks:4840"
```

Comment:
Don't use opcua.rocks here.  If someone generates a config file with opcua enabled we don't want to point telegraf there.  It might be good to default to localhost if you expect people to run telegraf on the same machine as their opcua endpoint or leave it unset so people will have to provide their endpoint for it to work.